yarn.lock

```diff
@@ -6436,7 +6436,7 @@ typedoc@0.22.17:
     minimatch "^5.1.0"
     shiki "^0.10.1"
 
-typescript@^4.1.3, typescript@^4.4.3:
+typescript@^4.1.3, typescript@^4.4.3, typescript@^4.7.4:
```

Comment:
Seems like several of the deps are now out of sync with what's in `@salesforce/dev-scripts`.  Just verifying that's the way to go.